components/emoji_picker/components/emoji_picker_header.jsx

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import * as Utils from 'utils/utils.jsx';
+
+export default class EmojiPickerHeader extends React.PureComponent {
+    static propTypes = {
+        handleEmojiPickerClose: PropTypes.func.isRequired,
+    };
+
+    render() {
+        return (
+            <div className='emoji-picker__header modal-header'>
+                <button
+                    type='button'
+                    className='close emoji-picker__header-close-button'
+                    onClick={this.props.handleEmojiPickerClose}
+                >
+                    <span aria-hidden='true'>{'×'}</span>
+                    <span className='sr-only'>{Utils.localizeMessage('emoji_picker.close', 'Close')}</span>
```

Comment:
Done. Question though, is there a way to pass a className to the FormatMessage's resulting <span> tag? I'd prefer to have only the one span tag rather than nested span tags just for a class. I haven't had a chance to look into the inner workings of FormatMessage yet, so thought I'd ask here first.

Reply:
Awesome, thanks for updating!

Reply:
I don't think there is - https://github.com/yahoo/react-intl/blob/master/src/components/message.js